dev/com.ibm.ws.com.unboundid/src/com/ibm/ws/com/unboundid/InMemoryLDAPServer.java

```diff
@@ -47,37 +57,66 @@
     private InMemoryDirectoryServerConfig config = null;
     private InMemoryDirectoryServer ds = null;
 
+    private static final String keystorePassword = "LDAPpassword";
+    private String keystore;
+
     /**
      * Creates a new instance of the in memory LDAP server. It initializes the directory
      * service.
      *
      * @param bases The base entries to create for this in-memory LDAP server.
      * @throws Exception If something went wrong
      */
-    public InMemoryLDAPServer(String... bases) throws Exception {
+    public InMemoryLDAPServer(boolean useWimSchema, boolean useSSLConfig, String... bases) throws Exception {
 
         config = new InMemoryDirectoryServerConfig(bases);
         config.addAdditionalBindCredentials(getBindDN(), getBindPassword());
-        config.setListenerConfigs(
-                                  InMemoryListenerConfig.createLDAPConfig("LDAP", // Listener name
-                                                                          null, // Listen address. (null = listen on all interfaces)
-                                                                          0, // Listen port (0 = automatically choose an available port)
-                                                                          null) // StartTLS factory
-        ); // Client factory
 
+        if (useSSLConfig) {
```

Comment:
I thought we talked about allowing both LDAP and LDAPS? We were going to add it to a collection and pass it into setListenerConfigs?

We would then probably have two ports and require changes to how we get the port (the method in this class).